src/components/QueryEditor/EditorField.tsx

```diff
@@ -2,22 +2,23 @@ import { css } from '@emotion/css';
 import React, { ComponentProps } from 'react';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { Space } from './Space';
-import { Field, Icon, PopoverContent, ReactUtils, stylesFactory, Tooltip, useTheme2 } from '@grafana/ui';
+import { Field, Icon, PopoverContent, ReactUtils, Space, Tooltip, useStyles2 } from '@grafana/ui';
 
 interface EditorFieldProps extends ComponentProps<typeof Field> {
   label: string;
   children: React.ReactElement;
   width?: number | string;
   optional?: boolean;
   tooltip?: PopoverContent;
+  tooltipInteractive?: boolean;
 }
-
+/**
+ * Uses Space component from grafana-ui. Available starting from grafana-ui@10.4.0
+ */
```

Comment:
Instead of adding comments next to each components, I would just make it clear in the main package `README.md` to write the min supported grafana version, which is in this case v10.4.